_episodes/01-intro.md

```diff
@@ -52,14 +52,14 @@ This means that, for example:
  - `weight` and `Weight` are different variables
 
 ## Types of data
-Python knows various types of data. Three common ones are:
+Python knows and can automatically recognize various types of data. Three common ones are:
```

Comment:
I understand what you're trying to say here but I think our learners will be left hanging as to what the phrase "automatically recognize various types of data" actually means.  If you think this is an important point, let's add it as a note (a sentence or two) to the end of this section (and undo this change here), something along the lines of "Note that we didn't have to tell Python the types of variables when we were creating them. Python automatically recognizes...".